framework/tests/src/bin/token_bridge.rs

```diff
@@ -0,0 +1,123 @@
+// This scripts helps create a test environment for sending native tokens over an IBC connection between 2 chains
```

Comment:
Lol, cw-orch + starship = OP

Reply:
it is ! Creating such testing evnironment was so easy using cw-orch !